app/views/incident_reports/_form.html.haml

```diff
@@ -56,7 +56,7 @@
       = form.label :speed
       = form.number_field :speed, id: :incident_report_speed, min: 0
     .field
-      = form.fields_for :incidents do |incident_form|
+      = form.fields_for :incidents, @incident do |incident_form|
```

Comment:
If I read the docs correctly, this could just be:

```suggestion
      = form.fields_for :incident do |incident_form|
```

Reply:
I get an association type mismatch error when I make this change.

Reply:
In the update action, I see.

In order for the association-based shorthand to work, `IncidentReport` also needs `accepts_nested_attributes_for :incident`. Not sure if that desired...?

Reply:
What would be the benefits of doing this?

Reply:
I mean, pretty much what you see here. It allows you to construct nested forms for associated objects without any specialized handling ([such as this](https://github.com/umts/incidents/blob/a775b7602ede59a89e3af4cb4bef7252f9ced039/app/controllers/incident_reports_controller.rb#L50-L52)). But the benefit is pretty limited for one or two fields, IMO.

It's especially neat for `has_many` relationships, because it will repeat the `fields_for` block for each one ([like this](https://github.com/umts/incidents/blob/a775b7602ede59a89e3af4cb4bef7252f9ced039/app/views/incident_reports/_form.html.haml#L310)).

I would say probably not worth it here, but up to whatever you prefer.

But... I *do* think the parameter key should be `incident` not `incidents` (since an incident report only has one incident).